### ISSUE TITLE
Metal LLVM IR

### ISSUE THREAD

**Issue by kvark** [gfx-rs/wgpu#4301]
Apparently, some parties ship compiled LLVM IR instead of MSL and it still works: https://twitter.com/icculus/status/721893213452312576

> We shipped using the LLVM bitcode they produce. We don't compile on the device. Don't know the details. "It just works"

**Comment by cwfitzgerald** [comment]
I don't think this is realistically in scope any time soon, maybe in a few years we can revisit if this would be interesting.
